### DISCUSSION TITLE
如何添加硕士在读期间研究成果部分

### DISCUSSION THREAD

**Discussion by Jerry-hyl** [stone-zeng/fduthesis#314] [Q&A]
求救，请问应该如何添加硕士在读期间研究成果部分呀？现在只能一次性输出参考文献列表，希望能够把研究成果文献列表作为第二部分在研究成果部分输出

**Comment by stone-zeng** [comment]
建议 `\backmatter` 之后直接加一章手动写，参考 #297。
